schema/rop.js

```diff
@@ -0,0 +1,53 @@
+const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
+
+const projectVersionStatuses = [
+  'draft',
+  'granted',
+  'submitted'
+];
+
+class Rops extends BaseModel {
+  static get tableName() {
+    return 'rops';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        id: { type: 'string', pattern: uuid.v4 },
+        projectId: { type: 'string', pattern: uuid.v4 },
+        status: { type: 'string', enum: projectVersionStatuses },
+        createdAt: { type: 'string', format: 'date-time' },
+        updatedAt: { type: 'string', format: 'date-time' },
+        deleted: { type: ['string', 'null'], format: 'date-time' }
+      }
+    };
+  }
+
+  static get relationMappings() {
+    return {
+      project: {
+        relation: this.BelongsToOneRelation,
+        modelClass: `${__dirname}/project`,
```

Comment:
We'll want to update the project schema to add a `HasMany` relation the other way too.

Reply:
you are right, I was more pushing this for confirming the fields